administrators.rst

```diff
@@ -680,16 +681,19 @@ Fields:
 * **Path**: *Not required: see below*.
 * **Staging path**: the absolute path to a staging area on the same server as
   the storage service.
+* **Bucket**: the name of the bucket to use. *Not required: see below*
```

Comment:
I think this just needs to rebase to pick up Ashley's addition here, https://github.com/artefactual/archivematica-storage-service-docs/commit/1281808bd43b435026700b44b1f2c8afdb069ff3